src/title/TitleStream.js

```diff
@@ -1,6 +1,6 @@
 /* @flow strict-local */
 
-import React, { PureComponent } from 'react';
+import React from 'react';
```

Comment:
```
I've chosen to wrap it in React.memo to mirror the usage of
PureComponent, although I'm not sure that we actually get a performance
benefit from doing so with this particular component.
```

Good thought!

Hmm, yeah, I don't see an obvious potential for an important performance benefit here.

Based on an earlier discussion, [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/converting.20to.20Hooks/near/1092351), I think this means we should do without `React.memo` here. Greg says (edited just so the issue link works):
> I think this will be a lot like `createSelector`: #3927. We'll want to introduce explicit memoization where there's significant computation happening in a render method -- either with `React.memo`, or perhaps with `useMemo` inside the component -- but because it makes the code more complicated, we'll want to do so only then, and not just all the time.

Reply:
Yeah, agreed.